Modules/_json.c

```diff
@@ -1866,25 +1814,59 @@ PyDoc_STRVAR(module_doc,
 static int
 _json_exec(PyObject *module)
 {
-    if (PyType_Ready(&PyScannerType) < 0) {
+    PyObject *PyScannerType = PyType_FromSpec(&PyScannerType_spec);
+    if (PyScannerType == NULL) {
         return -1;
     }
-    if (PyType_Ready(&PyEncoderType) < 0) {
+    get_json_state(module)->PyScannerType = PyScannerType;
+    Py_INCREF(PyScannerType);
+    if (PyModule_AddObject(module, "make_scanner", get_json_state(module)->PyScannerType) < 0) {
```

Comment:
`Py_DECREF((PyObject*)&PyScannerType);` is still needed here, no? Same remark for the similar code below.